service-chain-aws/deploy-scn-spn-sen/variables.tf

```diff
@@ -0,0 +1,129 @@
+variable "scn_instance_count" {
+  description = "The Number of SCN node"
+  default     = 4
+}
+
+variable "spn_instance_count" {
+  description = "The Number of SPN node"
+  default     = 2
+}
+
+variable "sen_instance_count" {
+  description = "The Number of SEN node"
+  default     = 2
+}
+
+variable "en_instance_count" {
+  description = "The Number of EN node"
+  default     = 1
+}
+
+variable "en_instance_type" {
+  description = "EN instance node type"
+  default     = "m5.2xlarge"
```

Comment:
If EN is for development environment, I think that 8 vCPU is over spec. If it is good to baobab or cypress, what about a comment?

Reply:
@henry-will I also think its a bit large for development, maybe 2 vCPU and 4GB of memory is enough for low-transaction development environment.. Need some more comments from Servicechain and Interface team.

Reply:
According to [Klaytn Docs](https://ko.docs.klaytn.com/node/endpoint-node/system-requirements), 8 vCPU is recommended. The test environment may differ to the production environment, however I'm worried that it may cause confusion when transitioning to production.